app.py

```diff
@@ -69,20 +71,124 @@ async def rather(ctx: Context, arg1: str = "normal"):
 
 
 @bot.command()
-async def quiz(ctx: Context, arg1: str = ""):
+async def trivia(ctx: Context, arg1: int = 5, arg2: int = 30):

```

Comment:
Add some notification mechanism if you want a timed delay before the trivia game starts. It would be good for mobile users.